packages/grid/x-grid-modules/src/components/styled-wrappers/grid-root.tsx

```diff
@@ -154,6 +154,13 @@ export const RootStyle = styled.div<GridRootProps>`
           &.checkbox-selection-header-cell .checkbox-input {
             padding: 12px;
           }
+          .dragging {
+            background: gray;
```

Comment:
Color should be in hex, and please try to reuse existing ones so it would be easier to identify when we refactor the theme properly